src/schema/anchor/playsByEpisode.json

```diff
@@ -0,0 +1,191 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Generated schema for Root",
+  "type": "object",
+  "properties": {
+    "provider": {
+      "type": "string"
+    },
+    "version": {
+      "type": "number"
+    },
+    "retrieved": {
+      "type": "string"
+    },
+    "meta": {
+      "type": "object",
+      "properties": {
+        "show": {
+          "type": "string"
+        },
+        "endpoint": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "show",
+        "endpoint"
+      ]
+    },
+    "range": {
+      "type": "object",
+      "properties": {
+        "start": {
+          "type": "string"
+        },
+        "end": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "start",
+        "end"
+      ]
+    },
+    "data": {
+      "type": "object",
+      "properties": {
+        "stationId": {
+          "type": "number"
+        },
+        "kind": {
+          "type": "string"
+        },
+        "parameters": {
+          "type": "object",
+          "properties": {
+            "timeRange": {
+              "type": "array",
+              "items": {
+                "type": "number"
+              }
+            },
+            "timeInterval": {
+              "type": "number"
+            },
+            "limit": {
+              "type": "number"
+            }
+          },
+          "required": [
+            "timeRange",
+            "timeInterval",
+            "limit"
+          ]
+        },
+        "data": {
+          "type": "object",
+          "properties": {
+            "rows": {
+              "type": "array",
+              "items": {
+                "type": "array",
+                "items": [
+                  {
+                    "type": "number"
+                  },
+                  {
+                    "type": "string"
+                  },
+                  {
+                    "type": "number"
+                  }
+                ],
+                "additionalItems": false
+              }
+            },
+            "columnHeaders": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string"
+                  },
+                  "type": {
+                    "type": "string"
+                  },
+                  "isDateTime": {
+                    "type": "boolean"
+                  },
+                  "labelKey": {
+                    "type": "string"
+                  }
+                },
+                "required": [
+                  "type"
+                ]
+              }
+            },
+            "labels": {
+              "type": "object",
+              "properties": {
+                "EpisodeTitle": {
+                  "type": "object",
+                  "properties": {
+                    "0": {
```

Comment:
this should be fixed but a generic index of type string as it can be another id as well, not just 0-7, right?

Reply:
true.